yml/imsim.yaml

```diff
@@ -8529,6 +8529,11 @@ tables:
     length: 16
     mysql:datatype: CHAR(16)
     description: id for truth_summary source.
+  - name: truth_type
+    '@id': '#MatchesTruth.truth_type'
+    datatype: long
+    mysql:datatype: BIGINT
```

Comment:
Regular `int` seems sufficient to hold values up to 3? Any other reason to make this a long/BIGINT? Same for `match_truth_type` below.